.npmrc

```diff
@@ -1 +1,2 @@
 auto-install-peers=true
+package-manager-strict=false
```

Comment:
pnpm 9 start to fail if you don't have the exact right version installed. I don't think we are ready to have everybody use corepack to manage that just yet so disabling for now.